test/e2e/suite/issuers/acme/certificate/http01.go

```diff
@@ -1,677 +0,0 @@
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-	It("should allow updating an existing certificate with a new dns name", func() {
-		certClient := f.CertManagerClientSet.CertmanagerV1().Certificates(f.Namespace.Name)
-
-		By("Creating a Certificate")
-		cert := gen.Certificate(certificateName,
-			gen.SetCertificateSecretName(certificateSecretName),
-			gen.SetCertificateIssuer(cmmeta.ObjectReference{Name: issuerName}),
-			gen.SetCertificateDNSNames(fmt.Sprintf("%s.%s", cmutil.RandStringRunes(5), acmeIngressDomain)),
-		)
-		cert.Namespace = f.Namespace.Name
-
-		_, err := certClient.Create(context.TODO(), cert, metav1.CreateOptions{})
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Waiting for the Certificate to be issued...")
-		err = f.Helper().WaitCertificateIssued(f.Namespace.Name, certificateName, time.Minute*5)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Validating the issued Certificate...")
-		err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Getting the latest version of the Certificate")
-		cert, err = certClient.Get(context.TODO(), certificateName, metav1.GetOptions{})
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Adding an additional dnsName to the Certificate")
-		newDNSName := fmt.Sprintf("%s.%s", cmutil.RandStringRunes(5), acmeIngressDomain)
-		cert.Spec.DNSNames = append(cert.Spec.DNSNames, newDNSName)
-
-		By("Updating the Certificate in the apiserver")
-		cert, err = certClient.Update(context.TODO(), cert, metav1.UpdateOptions{})
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Waiting for the Certificate to be not ready")
-		_, err = h.WaitForCertificateNotReady(f.Namespace.Name, certificateName, time.Minute*5)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Waiting for the Certificate to be issued...")
-		err = f.Helper().WaitCertificateIssued(f.Namespace.Name, certificateName, time.Minute*5)
-		Expect(err).NotTo(HaveOccurred())
-
-		By("Validating the issued Certificate...")
-		err = f.Helper().ValidateCertificate(f.Namespace.Name, certificateName, validations...)
-		Expect(err).NotTo(HaveOccurred())
-	})
-
-})
-
-// findDNSNames decodes and returns the dns names (SANs) contained in a
-// certificate secret.
-func findDNSNames(s *corev1.Secret) ([]string, error) {
-	if s.Data == nil {
-		return nil, fmt.Errorf("secret contains no data")
-	}
-	pkData := s.Data[corev1.TLSPrivateKeyKey]
-	certData := s.Data[corev1.TLSCertKey]
-	if len(pkData) == 0 || len(certData) == 0 {
-		return nil, fmt.Errorf("missing data in CA secret")
-	}
-	cert, err := tls.X509KeyPair(certData, pkData)
-	if err != nil {
-		return nil, fmt.Errorf("failed to parse data in CA secret: %w", err)
-	}
-
-	x509Cert, err := x509.ParseCertificate(cert.Certificate[0])
-	if err != nil {
-		return nil, fmt.Errorf("internal error parsing x509 certificate: %w", err)
-	}
-
-	return x509Cert.DNSNames, nil
-}
```

Comment:
Where did `test/e2e/suite/issuers/acme/certificate/http01.go` go? @wallrj

Reply:
https://github.com/jetstack/cert-manager/blob/c7514d92624613395eb00da8594ffa5391a0b169/test/e2e/suite/conformance/certificates/tests.go#L252-L278
https://github.com/jetstack/cert-manager/blob/c7514d92624613395eb00da8594ffa5391a0b169/test/e2e/suite/conformance/certificates/tests.go#L684-L729
...
Al of the removed tests were moved to `/test/e2e/suite/conformance/certificates/tests.go`, but some of these tests already were present there.

Reply:
Weird, `git` is unable to tell me about these moves 🤔 Have these moves been documented in commit messages?

Reply:
I didn't document what happened with each test individually. A lot of the tests did already overlap with the existing conformance tests. All tests that were not yet present in the conformance tests can be found in this PR as additions to the `/test/e2e/suite/conformance/certificates/tests.go` file. Sorry for the inconvenience. If there is something else I can still do to make this clearer, please let me know.